### PR TITLE
Query param driven sporks

### PR DESCRIPTION
**Pull request by gusvargas**
So that's it's easier for other rules to target resources processed by sporks.

### CODE REVIEW

File: packages/sporks-loader/src/index.js [hunk 1]
```diff
@@ -20,8 +17,6 @@ type Handler = (...args: Array<string>) => Promise<Array<WebpackModule>>;
 
 type Handlers = { [directive: string]: Handler };
 
-const loaders = internalLoaderPath;
-
 module.exports = function(source: string, sourceMap: any) {
   if (this._module.meta.source == null || !this._module.meta.directives) {
     // unlikely. something must have gone wrong in pitch
```
Comment (also) [inline packages/sporks-loader/src/index.js]: This is a lot more likely to happen now. Probably good to remove this comment.

File: packages/sporks-loader/src/index.js [hunk 2]
```diff
@@ -123,7 +118,7 @@ module.exports = function(source: string, sourceMap: any) {
                 'The require_env directive should take no parameters'
               );
             }
-            const { name, ext } = path.parse(mod.resource);
```
Comment (also) [inline packages/sporks-loader/src/index.js:126]: I don't think `mod` and `this` refer to the same module/context.

File: packages/sporks-loader/src/internal-loader.js [hunk 3]
```diff
@@ -9,11 +7,3 @@ module.exports = function(...args) {
   this._module.useSourceMap = true;
   this.callback(null, ...args);
 };
-
-module.exports.pitch = function() {
```
Comment (also) [inline packages/sporks-loader/src/internal-loader.js:13]: 😍

**Comment by gusvargas** [comment]
I went ahead and just fixed https://github.com/HubSpot/asset-bender/issues/12 here in https://github.com/HubSpot/asset-bender/pull/11/commits/7ce0a135562692e10f33a96564297e2b614706f4

**Review by also — APPROVED** [review]

**Pull request merged**